from exception.base_exception import BaseGuardException


class ArgumentNotInstanceOfException(BaseGuardException):
    pass
